xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/commentsinline.vm

```diff
@@ -312,6 +319,29 @@ $xwiki.ssfx.use('uicomponents/viewers/comments.css', true)
   </div>
 #end
 ##
+##
+##
+#macro(showDeleteModal)
+  <div class="modal fade" id="deleteModal" tabindex="-1" role="dialog">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal">&times;</button>
+          <h4 class="modal-title">$services.localization.render('core.viewers.comments.delete')</h4>
+        </div>
+        <div class="modal-body">
+          <div>$services.localization.render('core.viewers.comments.delete.confirm')</div>
+        </div>
+        <div class="modal-footer">
+          <input type="button" class="btn btn-danger"
+            value="$escapetool.xml($services.localization.render('core.viewers.comments.delete'))" data-dismiss="modal">
+          <input type="button" class="btn btn-default" value="$escapetool.xml($services.localization.render('cancel'))" data-dismiss="modal">
```

Comment:
This line exceeds 120 chars.

Reply:
I don't really see this as a rule for vm files.

Reply:
> I don't really see this as a rule for vm files.

I really think it should be a rule for any **code** we write, as otherwise it's a pain to read / review the code if you need to scroll horizontally.

> our existing VM and wiki pages are full for lines way bigger than 120 characters

We need to fix that.